config/routes.rb

```diff
@@ -174,5 +174,8 @@
   get "/422", to: "errors#unprocessable_entity", via: :all
   get "/500", to: "errors#internal_server_error", via: :all
 
+  mount OpenApi::Rswag::Ui::Engine => "/api-docs"
```

Comment:
_We could always stick the route behind `unless Rails.env.production?`_ (quote by Michal from slack) i.e.:

```diff
+ unless Rails.env.production?
+  mount OpenApi::Rswag::Ui::Engine => "/api-docs"
+  mount OpenApi::Rswag::Api::Engine => "/api-docs"
+ end
```

Reply:
Good point. We should probably think about whether we want to feature-flag this for production (and staging perhaps) or not while we're iterating on it. I completely can't make my mind up.

On the one hand we should hold it back until it's a bit more final.
On the other hand no-one will be looking at it in production and it won't be wrong just incomplete, so no consequence of shipping it.

Reply:
I would suggest doing that because the jQuery dependencies in that UI is massive behind the times and always get pulled out on a ITHC. Also even better to use the GOV UK Tech docs for consistency

Reply:
I think that's a decision to be made when we add the first api. Trade-offs should become more apparent then.

Reply:
But Matt raises good points, we don't want to cause R&P more trouble in their technical architecture review :P

Reply:
Where is the jQuery dependency sorry?

Reply:
Where was the jQuery dependency?

Reply:
looks like the UI is now React, so as long as the package pulls in the latest UI version we should be alright.